lib/structures/Invite.js

```diff
@@ -48,6 +50,8 @@ class Invite extends Base {
         this._createdAt = data.created_at !== undefined ? data.created_at : null;
         this.presenceCount = data.approximate_presence_count !== undefined ? data.approximate_presence_count : null;
         this.memberCount = data.approximate_member_count !== undefined ? data.approximate_member_count : null;
+        this.targetApplicationID = data.target_application_id !== undefined ? data.target_application_id : null;
```

Comment:
```suggestion
        this.targetApplicationID = data.target_application !== undefined ? data.target_application.id : null;
```
`target_application_id` is used when making an invite, use `target_application.id` instead.